grc/gui_qt/components/block_library.py

```diff
@@ -205,6 +207,9 @@ def __init__(self):
         # Dict representing which examples contain various blocks
         self.examples_w_block = {}
 
+        if not self.qsettings.value("appearance/display_blocklibrary", True, type=bool):
```

Comment:
What determines whether something goes under `appearance` or `grc`? It seems to be mix and match. Since there are `show_` vars under `grc`, it seems like these should go there. And they could be renamed `show_blocklibrary`, etc.

Reply:
Under appearance there is gui / widget related stuff. blocklibrary, console etc are dockwidgets. The show_..... are block or flowgraph related stuff. So I'd prefer to keep the setting  show/hidden for the dockwidgets under appearance.
By the way what about [grc] in config.conf?

Reply:
I deleted the `[grc]` section from `config.conf` on my machine and it was not added back in when I ran grc. Must have been old stuff.

Reply:
No, that's not old stuff. config.conf is never written by grc it's  readonly. 
This is a part of my config.conf

[grc]
canvas_default_size=1280, 1024
canvas_font_size=8
default_flow_graph=
editor=gedit
global_blocks_path=/usr/local/gnuradio/share/gnuradio/grc/blocks
local_blocks_path=/usr/local/oot3/share/gnuradio/grc/blocks
xterm_executable=/usr/bin/xterm

My oot's are located in local_blocks_path. If I remove config.conf  no oot's are found in grc --gtk.

If you remove config.conf  some settings may be taken from
/usr/local/gnuradio/etc/gnuradio/conf.d/grc.conf

But neither grc --gtk nor grc --qt has an editor for config.conf.

Reply:
So, none of these things will be read from `grc[_qt].conf`? What creates the `[grc]` entries in `config.conf`?

Reply:
At least xterm_executable is read from grc_qt.conf.
You have to create the entries in config.conf by yourself using vi or nano.
So I would plead for moving  the grc settings for grc --qt to grc_qt.conf  but keep them in config.conf  for the use in --gtk.
Doing so, we could edit these settings using the ( new ) preference editor.
But I'd like to do it in a new pr